client/scss/components/_preview-panel.scss

```
@@ -33,6 +34,10 @@
     transform: scale(var(--preview-width-ratio));
     display: block;
 
+    &:empty {
```

```suggestion
    &:empty {
      // Ensure that sites without a background show with a fallback, only when iframe has loaded
```

Same here. A comment to help explain this would be useful.